examples/stm32/f3/stm32f3-discovery/traceswo/README.md

```diff
@@ -0,0 +1,9 @@
+# README
+
+This is an example which uses the ITM and TPIU to print 
+debug data to the TRACESWO pin (PB3)
+
+It's intended for the ST STM32F3DISCOVERY eval board. 
+Pin PB3 will output strings in UART compatible format at a baudrate of 115200
```

Comment:
I feel you may be misunderstanding the purpose of trace, or the ways of using it.  Manually setting up the trace blocks, and using such a low baud rate is of marginal interest as a "look, I got one more free tx-only uart, on fixed pin" but is very much _not_ an example of "how to do trace"  You normally want to let your debug tool configure this, based on what you actually want to debug, not be some static setup.  Please read all the comments in https://github.com/libopencm3/libopencm3/pull/1054 for more insight, and feel very welcome to stop by the IRC channel if you want to discuss it further.